demo/protobuf_getting_started/update.sh

```diff
@@ -5,14 +5,18 @@ clean() {
   rm -rf deps
 }
 
+if [ -z ${SRCDIR} ]; then
```

Comment:
I'm curious if using `dirname "$0"/../..` would be slightly more robust in case there are multiple directories within a GOPATH: https://stackoverflow.com/questions/36017724/can-i-have-multiple-gopath-directories

Reply:
I mean for the default case.

Reply:
Good question. I wasn't even aware that you can have >1 path. Let me send a PR to fix this.